src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using System;
-using System.Collections;
 using System.Collections.Generic;
-using System.Globalization;
 using System.Linq;
 using System.Net;
 using System.Threading.Tasks;
+using AutoMapper.Execution;
```

Comment:
Can we use this or lose it? (Actually, just lose it - don't think it's used anywhere, maybe carried over from Employer solution).

Reply:
Done